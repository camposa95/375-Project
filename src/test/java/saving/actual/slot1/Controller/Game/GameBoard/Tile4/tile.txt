TerrainType: HILLS
DieNumber: 6
HasRobber: false
VertexIDs: [9, 10, 11, 19, 20, 21]
