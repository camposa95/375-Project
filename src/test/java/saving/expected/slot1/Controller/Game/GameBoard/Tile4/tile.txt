TerrainType: FORREST
DieNumber: 4
HasRobber: false
VertexIDs: [9, 10, 11, 19, 20, 21]
